gitlab_migration.md

```diff
@@ -74,7 +74,7 @@ Go to Settings -> CI/CD:
 
 Examples:
 ```
-/gitlab ucsdlibrary/apbatch run staging_deploy
+/gitlab ucsdlibrary/apbatch run staging_deploy 1.5.0
```

Comment:
@mcritchlow - what's that number 1.5.0? Is it a tag? Thanks

Reply:
@VivianChu - yep that's a tag. Should I add a note about that? Perhaps so, since you needed to ask :)

Reply:
@VivianChu - I've updated the branch to try and clarify. Can you take another look when you have a chance and let me know if that's better?